akka-diagnostics/src/main/scala/akka/diagnostics/ConfigChecker.scala

```diff
@@ -807,8 +894,9 @@ class ConfigChecker(system: ExtendedActorSystem, config: Config, reference: Conf
       Vector.empty[ConfigWarning] ++
       checkAutoDown() ++
       checkClusterFailureDetector() ++
-      checkClusterDispatcher()
-    } else Vector.empty[ConfigWarning]
+      checkClusterDispatcher() ++
+      checkCreateActorRemotely()
+    } else Vector.empty[ConfigWarning] ++ checkPreferClusterThanRemote()
```

Comment:
better to move this `checkPreferClusterThanRemote` to `checkRemote`

Reply:
yep. Done.